docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
```

Comment:
I recommend to use version 2 the docker-compose. It improve the compatibility with various version the docker-compose. Version 2 is compatible with 1.9+.